plugins/labhub.py

```diff
@@ -196,7 +196,7 @@ def unassign_cmd(self, msg, match):
             else:
                 return 'You are not an assignee on the issue.'
 
-    @re_botcmd(pattern=r'mark\s+(wip|pending)\s+https://(github|gitlab)\.com/([^/]+)/([^/]+)/(pull|merge_requests)/(\d+)',  # Ignore LineLengthBear, PyCodeStyleBear
+    @re_botcmd(pattern=r'mark\s+(wip|pending(?:[-\s]+review)?\b)\s+https://(github|gitlab)\.com/([^/]+)/([^/]+)/(pull|merge_requests)/(\d+)',  # Ignore LineLengthBear, PyCodeStyleBear
```

Comment:
Wrong, we will not use - and space together.

Reply:
Could you help me on the tests? I'm not using captured groups, but the tests are still failing

Reply:
Edited my comment, see the regex

Reply:
Just asking for my understanding. Anything inside the square bracket would mean choose anyone one of the available options. So, here it would be either - or space. 
So, why is it not advised to use the two together?